src/raft/uv_segment.c

```diff
@@ -729,13 +729,14 @@ int uvSegmentBufferFormat(struct uvSegmentBuffer *b)
 	}
 	b->n = n;
 	cursor = b->arena.base;
-	bytePut64(&cursor, UV__DISK_FORMAT);
+	bytePut64(&cursor, format_version);
```

Comment:
Should we carry the format version in the segment itself?

Reply:
Not sure what you mean---keep it as a field in `struct uvSegmentBuffer`?

Reply:
Precisely that

Reply:
Yeah, good point, this is more consistent with how we pass in `uv->block_size`.

Reply:
I see that there is a new field now in `uvSegmentBuffer` but it is not consistently used, e.g. `uvSegmentBufferAppend` takes the format_version as an argument an uses that without accessing the one in the struct.